twisted/topfiles/7957.feature

```diff
@@ -0,0 +1,3 @@
+twisted.python.syslog has been ported to Python 3.
+twisted.python.test.test_syslog runs under Python 3.
```

Comment:
the twisted.python.test is our private place so we should not talk about it in the release notes.